File: WebShop/Dto/OrderDto.cs
using WebShop.Enums;
using WebShop.Models;

namespace WebShop.Dto
{
    public class OrderDto
    {

        public string Id { get; set; }

        public string UserId { get; set; }

        public string ShippingAddress { get; set; }

        public List<ItemDto> OrderedItems { get; set; }

        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }

        public int DeliveryPrice { get; set; }
        public int Price { get; set; }

        public OrderStatus OrderStatus { get; set; }
    }
}

File: frontend/webshop/src/models/OrderDto.js
export default class OrderDto {
    constructor(Id, OrderedItems, userId, deliveryAddress, price) {
        this.Id = Id;
        this.OrderedItems = OrderedItems;
        this.UserId = userId;
        this.ShippingAddress = deliveryAddress;
        this.Price = price;
        
    }

    addArticle(OrderedItems) {
        this.OrderedItems.push(OrderedItems);
    }
}